packages/example/package.json

```diff
@@ -12,6 +12,8 @@
   "license": "MIT",
   "dependencies": {
     "@celo-tools/use-contractkit": "1.0.6",
+    "@celo/contractkit": "^1.2.4",
```

Comment:
Would  be good to use the same ck version in the example as in the use-contractkit package.json
How about we upgrade it there?

Reply:
They're both 1.2.4 already.